packages/core/streams/ai-stream.ts

```diff
@@ -16,7 +16,7 @@ export interface ToolCallPayload {
     type: 'function';
     func: {
       name: string;
-      arguments: Record<string, unknown>;
+      arguments: string;
```

Comment:
This impacts non-RSC code. Have you tested this with any of the regular function calling examples?

Reply:
Also, why does this only affect tool calls but not function call (line 11)?